dev/Materials/Acrylic/AcrylicBrush_19h1_themeresources.xaml

```diff
@@ -28,22 +28,48 @@
                 TintOpacity="0.0"
                 FallbackColor="#F9F9F9"
                 BackgroundSource="HostBackdrop"/>
+            <local:AcrylicBrush
+                x:Key="AcrylicInAppFillColorDefaultInverseBrush"
```

Comment:
TintLuminosityOpacity is set in code due to a markup limitation. Do you need to set it for these set of brushes as well ? See here - [https://github.com/microsoft/microsoft-ui-xaml/blob/08e5ac41f80b5c5f9b7185acb846615c3aa72b40/dev/dll/XamlControlsResources.cpp#L143](https://github.com/microsoft/microsoft-ui-xaml/blob/08e5ac41f80b5c5f9b7185acb846615c3aa72b40/dev/dll/XamlControlsResources.cpp#L143)

Reply:
Yes, see line 149 in XamlControlsResources.cpp file in this PR.

`if (const auto acrylicInAppFillColorDefaultInverseBrush = dictionary.Lookup(box_value(c_AcrylicInAppFillColorDefaultInverseBrush)).try_as<winrt::AcrylicBrush>())
    {
        acrylicInAppFillColorDefaultInverseBrush.TintLuminosityOpacity(0.96);
    }`